content/en/Integrations/webhooks.md

```diff
@@ -12,7 +12,7 @@ description: >
 Configure webhooks to subscribe to real-time updates for your pentests.
 {{% /pageinfo %}}
 
-With our API-based webhooks, you can set up an integration between your app and the Cobalt platform to get notifications for pentest events. We’ll send you updates for each event to your URL through an HTTP POST request.
+With our API-based webhooks, you can set up an integration between other apps and the Cobalt platform to get notifications for pentest events. We’ll send updates for each event to user-configured, secure URLs through an HTTP POST request.
```

Comment:
@nktrk 

> other apps

We write documentation from the [second-person point of view](https://design.gitlab.com/content/voice-tone#point-of-view). Most likely, a developer or admin will configure webhooks for the application in their organization, so it's simply _their_ app.

> user-configured, secure URLs

Same reasoning as in the previous comment. If we want to add _secure_ for a URL, I would do it in the _Before Your Start_ section:

Make sure that the **URL** where you want to receive notifications is valid and secure. If possible, test your connections and check that your services work properly.